front/src/common/api/generatedEditoastApi.ts

```diff
@@ -811,6 +811,17 @@ const injectedRtkApi = api
         }),
         invalidatesTags: ['work_schedules'],
       }),
+      postWorkSchedulesProjectPath: build.mutation<
```

Comment:
If I understand correctly, it's a post request because of the body but we will use it like a "get".
If that's the case, we should add a special rule for this endpoint like we did for others in the file `openapi-editoast-config.ts`.

Reply:
good point, changed it